src/contour/display/TerminalDisplay.h

```diff
@@ -36,8 +36,6 @@ namespace contour::display
 
 class OpenGLRenderer;
 
-// It currently just handles one terminal inside, but ideally later it can handle
-// multiple terminals in tabbed views as well tiled.
 class TerminalDisplay: public QQuickItem
```

Comment:
Why did you drop the comment entirely?

Reply:
yes, suppose should adjust it to show what is actually happening with the display now